README.md

```diff
@@ -322,6 +323,27 @@ typedef NS_ENUM(NSInteger, NYTAdRequestState) {
 };
 ```
 
+## Boxing Enumerated Types
+
+Although `enum`s are integral values, they cannot be directly used as [boxed literals](http://clang.llvm.org/docs/ObjectiveCLiterals.html#boxed-enums). When boxing `enum`s (for example when using them in a Cocoa collection) they should be boxed as `NSNumber`s using the shorthand `@()` literal expression, not as the `[NSNumber numberWith...:]` methods.
+
+This eliminates the need to look up the underlying type of the enum in order to box it correctly.
+
+**For example:**
+```objc
+typedef NS_ENUM(NSInteger, NYTDaysOfTheWeek) {
```

Comment:
@spacedrabbit Check out [Enumerations on the "Using Swift with Cocoa and Objective-C" docs](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/BuildingCocoaApps/InteractingWithCAPIs.html#//apple_ref/doc/uid/TP40014216-CH8-XID_13).